tests_e2e/orchestrator/runbook.yml

```diff
@@ -22,9 +21,28 @@ variable:
   - name: admin_password
     value: ""
     is_secret: true
+  - name: keep_environment
+    value: "no"
```

Comment:
Is this just a default value? Will this get replaced by the value for keep_environment from the azure pipeline? I dont see where we update the value

Reply:
yes, this is only the default value, good question.

these parameters get overwritten with the values from the pipeline when we invoke lisa from execute_tests.sh

      "lisa \
          --runbook \$HOME/WALinuxAgent/tests_e2e/orchestrator/runbook.yml \
          --log_path \$HOME/logs/lisa \
          --working_path \$HOME/logs/lisa \
          -v subscription_id:$SUBSCRIPTION_ID \
          -v identity_file:\$HOME/.ssh/id_rsa \
          -v test_suites:\"$TEST_SUITES\" \
          -v collect_logs:\"$COLLECT_LOGS\" \
          -v keep_environment:\"$KEEP_ENVIRONMENT\**"" \